packages/types/src/networks.ts

```diff
@@ -171,4 +173,6 @@ export enum CoingeckoPlatform {
   Story = "story",
   Ink = "ink",
   Taraxa = "taraxa",
+  UnitZero = "unit-zero",
```

Comment:
These seems to not be in the coingecko platform api. Please make sure you added the right platform or remove if there isn't one. Here's the platform api for reference: https://api.coingecko.com/api/v3/coins/list?include_platform=true

Reply:
yes, you are right. The correct value is "units-network". Fixed it